--- a/meson.build
+++ b/meson.build
@@ -1091,8 +1091,7 @@
     gio = not_found
   endif
   if gio.found()
-    gdbus_codegen = find_program('gdbus-codegen',
-                                 required: get_option('gio'))
+    gdbus_codegen = find_program('gdbus-codegen')
     gio_unix = dependency('gio-unix-2.0', required: get_option('gio'),
                           method: 'pkg-config')
     gio = declare_dependency(dependencies: [gio, gio_unix],
